src/main/java/core/basesyntax/service/impl/SupplyAction.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.service.ActionHandler;
+
+public class SupplyAction implements ActionHandler {
+    public void apply(String fruit, int quantity) {
```

Comment:
The method `apply` should be overridden from the `ActionHandler` interface. Please ensure to use the `@Override` annotation to make it clear that you're overriding a method.